common/beacon/client.go

```diff
@@ -19,7 +20,7 @@ func NewBeaconClient(ctx context.Context, cfg flags.BeaconConfig) (Client, error
 	cctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	c, err := http.New(cctx, http.WithAddress(cfg.BeaconURL), http.WithTimeout(cfg.BeaconClientTimeout))
+	c, err := http.New(cctx, http.WithAddress(cfg.BeaconURL), http.WithTimeout(cfg.BeaconClientTimeout), http.WithEnforceJSON(cfg.EnforceJSON), http.WithLogLevel(zerolog.ErrorLevel))
```

Comment:
nit: should we make the log level configurable? is error level enough?

Reply:
The logs that this client outputs are mostly duplicated with the service logs (e.g. [here](https://github.com/base-org/blob-archiver/blob/master/archiver/service/archiver.go#L83-L112)).

It also don't hook into oplog and as such don't format nicely -- so I figured it was worth preferring the services logs over the libraries.

That being said -- I don't have a strong opinion here. Happy to keep them in if you prefer.

Reply:
I trust your judgement here. thanks!